docs/discovery.md

```diff
@@ -116,11 +116,11 @@ You can list the nodes in your cluster.
 On each of your nodes, start the Swarm agent. The node IP address
 doesn't need to be public as long as the Swarm manager can access it.
 
-    swarm join --advertise=<node_ip:2375> consul://<consul_addr>/<optional path prefix>
+    swarm join --advertise=<node_ip:2375> consul://<consul_addr1>,<consul_addr2>/<optional path prefix>
```

Comment:
@thaJeztah that's actually a nice idea but CLI docs sort of like this, I guess we should either change both or none.

![image](https://cloud.githubusercontent.com/assets/159209/8891682/d4db0b1e-32e9-11e5-845d-ce3bae30108e.png)